tests/models/Location2.php

```diff
@@ -0,0 +1,10 @@
+<?php
+class Location2 extends Doctrine_Record
+{
+    public function setTableDefinition()
+    {
+        $this->hasColumn('id', 'integer', 10, array('primary' => true));
+        $this->hasColumn('lat', 'double', 10, array ());
+        $this->hasColumn('lon', 'double', 10, array ());
```

Comment:
```suggestion
        $this->hasColumn('lat', 'double', 10, array());
        $this->hasColumn('lon', 'double', 10, array());
```

Reply:
Do we want phpcs github action and/or a project based phpcs config?
Something like that:
```
- name: Setup PHP
  uses: shivammathur/setup-php@v2
  with:
    php-version: '8.1'
    tools: cs2pr, phpcs

- name: Run phpcs
  run: phpcs -q --report=checkstyle src | cs2pr
```